action/envelope.go

```diff
@@ -140,7 +140,11 @@ func (elp *envelope) LoadProto(pbAct *iotextypes.ActionCore) error {
 	elp.nonce = pbAct.GetNonce()
 	elp.gasLimit = pbAct.GetGasLimit()
 	elp.gasPrice = &big.Int{}
-	elp.gasPrice.SetString(pbAct.GetGasPrice(), 10)
+	_, ok := elp.gasPrice.SetString(pbAct.GetGasPrice(), 10)
+	// elp gas price gets zero when pbAct.GetGasPrice is empty string
```

Comment:
Sometimes, it's better to explain with code instead of comment
```
if pbAct.GetGasPrice() == "" {
	elp.gasPrice = big.NewInt(0)
} else {
	elp.gasPrice, ok = new(big.Int).SetString(pbAct.GetGasPrice(), 10);
	if !ok {
	}
}
```